### PR TITLE
[MRG+1] FIX markevery only accepts builtin integers, not numpy integers

### PR DESCRIPTION
**Pull request by kalagau**
This is the PR for #8057 
markevery now accepts numpy integers

### CODE REVIEW

**Comment by anntzer** [comment]
`isinstance(markevery, (int, np.integer))`

File: lib/matplotlib/lines.py [hunk 1]
```diff
@@ -143,6 +143,9 @@ def _slice_or_none(in_v, slc):
     # if just an int, assume starting at 0 and make a tuple
     elif isinstance(markevery, int):
         markevery = (0, markevery)
+    # if just an numpy int, assume starting at 0 and make a tuple
```
Comment (anntzer) [inline lib/matplotlib/lines.py]: ... and remove the lines above

Comment (kalagau) [inline lib/matplotlib/lines.py]: Wont this cause an issue for actual integers?
Because 4.items() would cause an error

Comment (anntzer) [inline lib/matplotlib/lines.py]: Good point, but then it should be isinstance(markevery, np.integer) below.

**Comment by kalagau** [comment]
Hmm interesting, travis CI build decides to fail for 3.4

**Comment by anntzer** [comment]
it's a well known flaky test that can be ignored

**Pull request merged**